content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -38,9 +38,11 @@ Configurations with a single API server will experience unavailability while the
    Any service account created after this point will get secrets that include both old and new CAs.
 
    {{< note >}}
-   Remove the flag `--client-ca-file` from the *Kubernetes controller manager* configuration.
-   You can also replace the existing client CA file or change this configuration item to reference a new, updated CA.
-   [Issue 1350](https://github.com/kubernetes/kubeadm/issues/1350) tracks an issue with *Kubernetes controller manager* being unable to accept a CA bundle.
+   The files specified by the *Kubernetes controller manager* flags `--client-ca-file` and `--cluster-signing-cert-file`
```

Comment:
nit: The page could use, kube-controller-manager instead of *Kubernetes controller manager*?
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

Reply:
added a second commit to use kube-controller-manager + glossary.